back-end/tests/loans.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require('supertest');
+const app = require('../app');
+
+describe('GET /loans', () => {
+    describe('Vaild request', () => {
```

Comment:
@puteulanus does this pass the linter? there should be 2 spaced for indentation, or there are for the other files